src/components/Send.vue

```diff
@@ -148,6 +148,15 @@ export default {
     Alert,
   },
   data() {
+    const enoughFunds = (rule, value, callback) => {
```

Comment:
What's `rule` here?

Reply:
This function is used by `Element` to define [custom validation rules](https://element.eleme.io/#/en-US/component/form#custom-validation-rules). `rule` contains information about the validations that you specified in the template such as the type or the the validator function for that type.